test/app/plugins.js

```diff
@@ -0,0 +1,21 @@
+require('../setup');
+
+const {_toDependencies} = require('../../app/plugins');
+
+describe('plugins', function () {
+  describe('#toDependencies()', function () {
+    it('should convert dependencies form hyperterm\'s format to npm\'s', function () {
+      const plugins = ['project1', 'project2#1.0.0', 'project3@beta',
+                       '@org1/project4#1.0.0', '@org2/project5@alpha',
+                       '@org3/project6'];
+
+      const result = _toDependencies({plugins: plugins});
+      result.should.have.property('project1', 'latest');
+      result.should.have.property('project2', '1.0.0');
+      result.should.have.property('project3', 'beta');
+      result.should.have.property('@org1/project4', '1.0.0');
+      result.should.have.property('@org2/project5', 'alpha');
+      result.should.have.property('@org3/project6', 'latest');
```

Comment:
`to satisfy` has a whole bag of tricks beyond this, but even in it's simplest form it's really useful :) The documentation site has a lot of examples and [a list of all the assertions](http://unexpected.js.org/assertions/any/to-be/) in core, plus [a list of plugins](http://unexpected.js.org/plugins/) (most of which also has documentation sites) that might come in useful when doing more specialized things.

I'm on the [unexpected gitter channel](https://gitter.im/unexpectedjs/unexpected) and I'd love to help you out if you have any questions :-)